Necessary for `timeval`, `FD_ZERO`, `FD_SET` and `select` to be available.
diff --git a/src/melt/io.c b/src/melt/io.c
index 7f6859d..9004a17 100644
--- a/src/melt/io.c
+++ b/src/melt/io.c
@@ -26,6 +26,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/time.h>
+#include <sys/select.h>
 #ifndef _WIN32
 #include <termios.h>
 #include <unistd.h>
